rancher/elemental-ui
# Install elemental-operator from Rancher Markeplace chart

Fixes #139 

- Use `v-clean-html` instead of `v-html`
- Bump shell version used to `0.3.27`
- Breakdown of `pkg/elemental/pages/index.vue` main view into components such as `DashboardView` and `InstallView`
- Create `InstallView` which enables user to install Elemental Operator from Rancher Marketplace

**To test:**
- You will need to have Rancher backend version `2.8.0` (2.8-head, 2.8.0-rc3, etc)
- Make sure you don't have already installed elemental operator
- Run this branch and go to elemental extension and follow the steps

**Preview:**
https://github.com/rancher/elemental-ui/assets/97888974/6b8781de-80ea-4b92-8d82-14994869ea8c
